templates/plugin-info/features.php

```diff
@@ -27,7 +27,9 @@
 					$features_plan_map[ $feature->id ] = array( 'feature' => $feature, 'plans' => array() );
 				}
 
-				$features_plan_map[ $feature->id ]['plans'][ $plan->id ] = $feature;
+                if ( ! empty( $plan->id ) ) {
```

Comment:
What if you add here

![CleanShot 2024-05-13 at 19 41 07@2x](https://github.com/Freemius/wordpress-sdk/assets/1623381/bc4ccbf3-1ca1-42c6-83b3-2d2f8ed057ed)

```php
/**
 * @var Plan[] $plans
 */
```

does it still need this empty check?